### DISCUSSION TITLE
Ability to individually save a message in a channel

### DISCUSSION THREAD

**Discussion by CornLuck** [stoatchat/discussions#533] [Feature Suggestions]
In discord people with specific perms can pin a message for everyone to return back to, but that feature would be really useful if you could also clientside save a message in a channel just for yourself, without having to pin it for everyone. I dont mean for this to override the pin feature, just have it work alongside it

**Comment by MasicoreLord** [comment]
As long as the personal pin disappears when the message itself does, I'm okay with that.
